openshift/e2e-tests-openshift.sh

```diff
@@ -14,7 +14,8 @@ readonly INSECURE="${INSECURE:-"false"}"
 readonly TEST_NAMESPACE=serving-tests
 readonly TEST_NAMESPACE_ALT=serving-tests-alt
 readonly SERVING_NAMESPACE=knative-serving
-readonly SERVICEMESH_NAMESPACE=istio-system
+readonly SERVICEMESH_NAMESPACE=knative-serving-ingress
+export GATEWAY_NAMESPACE_OVERRIDE="$SERVICEMESH_NAMESPACE"
```

Comment:
Where is this used?

Reply:
Knative's tests use this to determine the correct gateway.